### PR TITLE
New plots and information added in the html report

### PR DESCRIPTION
**Pull request by phupe**
New plots and information are provided in the html report in new tabs:

CPU:
* CPU Time
* Summary

Memory:
* RAM per core 
* Total RAM
* Summary

Job Execution (was named Job Duration intially):
* Summary

I/O:
* Total Read
* Total Write
* Summary

Other changes:
* plotly.js updated to v1.44.3

### CODE REVIEW

**Comment by pditommaso** [comment]
Oh, could you attached a screenshot of the resulting report?

**Comment by phupe** [comment]
screenshots added

cpu_cputime
cpu_summary
io_summary
io_totalread
io_totalwrite
job_summary
mem_mempercore
mem_summary
mem_totalmem

**Comment by pditommaso** [comment]
Why boxplots have bee replaced by a histograms? I think the first are more informative.

**Comment by phupe** [comment]
Actually, all the boxplots are still present since they are very informative as you mentioned.

In the tabs I have added, I used barplot as there is only one value to display. Indeed:
* CPU Time is the \sum_i cputime_of(task_i) (for each process)
* IO Total Write is the \sum_i data_written_by(task_i) (for each process)
* IO Total Read is the \sum_i data_read_by(task_i) (for each process)
* Total RAM is the \sum_i peak_rss_used_by(task_i) (for each process)

In other words, they are the cumulative sum over all the values of the tasks within the same process. The aim was to provide global statistics. In that way, the barplot provides a complementary information with respect to the boxplot.

The only exception is:
* RAM per core: the boxplot displays the average but I agree that I could use a boxplot instead.

**Comment by phupe** [comment]
I did some modification in the RAM per core tab: barplot has been replaced by a boxplot.

**Comment by phupe** [comment]
@pditommaso: do you suggest any other improvments, modifications?

**Comment by pditommaso** [comment]
Hi sorry, for the late reply. I will try to review asap.

**Comment by phupe** [comment]
great. thanks a lot.

**Comment by phupe** [comment]
Hi Paolo,

By chance, did you have time to have a look at the additional plots I proposed?
They would be really useful to investigate how the computing resources are used within a workflow.

Cheers

**Comment by pditommaso** [comment]
Apology for the very long delay. The main concern about this PR is that the logic is implemented on the client-side (ie. the report adding yet another loop over the tasks list) which makes the rendering even slower when the report contain many tasks. I think it should be possible to implement this at NF level as for the other metrics.

**Comment by pditommaso** [comment]
Yes, indeed they are useful and some of them have been included in the stats provided by Tower. Have you looked at it? Also, the idea is to reply more on Tower to manage these kinds of data because it allows better, tacking, storage and scalability. This is why I would prefer to focus on that.

**Comment by phupe** [comment]
I knew you work on a new tool for metric reporting but didn't know it was tower.
I will have a look at https://github.com/seqeralabs/nf-tower
Thanks

**Comment by pditommaso** [comment]
The plan is to implement this feature as part of Tower https://github.com/nextflow-io/nextflow/pull/1046. Therefore I'm closing this. Thanks for the PR and sorry for slow interaction.
